src/components/custom/_line-tab.js

```diff
@@ -0,0 +1,92 @@
+import React, { Children, useEffect, useState } from 'react'
```

Comment:
i dont see this _line tab component used anywhere, may i know where are we using this?

Reply:
@mustofa-binary we are using it in the new trustpilot section. It is planned to have 3 different tabs in the future until approval. For now it is there but won't show up any clickable navigations coz it is just 1 item.